zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebePriorityDefinition.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.model.bpmn.instance.zeebe;
+
+import io.camunda.zeebe.model.bpmn.instance.BpmnModelElementInstance;
+
+public interface ZeebePriorityDefinition extends BpmnModelElementInstance {
+
+  String DEFAULT_PRIORITY = "50";
```

Comment:
Good question, @mustafadagher 👍 @panagiotisgts, it could be beneficial to add a link to the design/implementation decisions that were made so reviewers can read up on them. You can link them from the PR and/or the issue so reviewers can discover them.

Reply:
Added the reference kick-off decisions comment for the feature in the description.